Sources/ComposableArchitecture/Documentation.docc/Articles/SharingState.md

```diff
@@ -33,6 +33,7 @@ system, such as SQLite.
 * [Observing changes to shared state](#Observing-changes-to-shared-state)
 * [Initialization rules](#Initialization-rules)
 * [Deriving shared state](#Deriving-shared-state)
+* [Concurrent mutations to shared state](#Concurrent-mutations-to-shared-state)
```

Comment:
I'm moving this up higher because it's important to know this before the testing material.